internal/lock_test.go

```diff
@@ -28,12 +28,30 @@ func TestNewLockValid(t *testing.T) {
 `)
 	lock, err := NewLock(path, false)
 	assert.Nil(t, err)
+
 	assert.Equal(t, 1, len(lock.conf.Resource))
 	statement := lock.conf.Resource[0]
 	assert.Equal(t, "sha256-asdasdasd", statement.Integrity)
 	assert.Equal(t, []string{"tag1", "tag2"}, statement.Tags)
+
+	runtime.GC() // Release resources
 }
 
+func TestDuplicateResource(t *testing.T) {
```

Comment:
Why are you moving this around and slightly modifying it? I don't see why you're doing this or the need for it.